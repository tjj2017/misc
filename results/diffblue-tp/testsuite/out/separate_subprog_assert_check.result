XFAIL:Assert statement in p.adb should succeed.
