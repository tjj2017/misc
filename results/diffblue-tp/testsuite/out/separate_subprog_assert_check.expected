[2] file p-inc.asu line 6 assertion: SUCCESS
[1] file p.adb line 16 assertion: SUCCESS
VERIFICATION SUCCEEDED
